cvpy/tests/test_image.py

```diff
@@ -24,7 +24,11 @@
 import sys
 import numpy as np
 from cvpy.base.ImageTable import ImageTable
-from cvpy.image.Image import Image
+# from cvpy.image.Image import Image
+from cvpy.image.NaturalImageTable import NaturalImageTable
+from cvpy.utils.ImageUtils import ImageUtils
+from cvpy.biomedimage.BiomedImage import BiomedImage
+
```

Comment:
Create two new files: test_image_utils.py and test_natural_imagetable.py.

Add all NaturalImageTable test cases to test_natural_imagetable.py file. 
Add all ImageUtils test cases to test_image_utils.py file. 

Remove any BiomedImage related tests from either of these file.